schema/m3-requirements.schema.json

```diff
@@ -424,6 +424,13 @@
                 "minimum": 0,
                 "title": "Shinespark Frames",
                 "description": "The duration (in frames) of the shinespark that must be executed. This defines the energy cost of the shinespark."
+              },
+              "excessShinesparkFrames": {
+                "$id": "#/definitions/logicalRequirement/items/properties/canComeInCharged/properties/excessShinesparkFrames",
+                "type": "integer",
+                "minimum": 0,
+                "title": "Excess Shinespark Frames",
+                "description": "The shinespark duration (in frames) that is not required to complete the objective of the shinespark.  With shinesparkFrames as the expected cost, this defines the lower limit of the shinespark energy cost."
```

Comment:
I think the second sentence here could be replaced with something like ...
"Subtracting this from the shinesparkFrames defines the lower limit of the shinespark energy cost."